invenio_rdm_records/fixtures/data/vocabularies.yaml

```diff
@@ -11,3 +11,6 @@ resource_types:
   pid-type: rsrct
   data-file: vocabularies/resource_types.yaml
 # 'subjects' is highly dependent on the kind of InvenioRDM instance, so no defaults are provided.
+subjects:
+  pid-type: subj
```

Comment:
We've been using `sub` in tests so far: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/fixtures/app_data/vocabularies.yaml#L4 . We should align with the same pid-type for subjects everywhere since external repositories will have to use that pid-type too (e.g. (and then I used it in external repositories https://github.com/fenekku/invenio-subjects-mesh/blob/master/invenio_subjects_mesh/vocabularies/vocabularies.yaml#L2).